src/main/java/uk/gov/hmcts/reform/divorce/orchestration/tasks/SendPetitionerClarificationRequestEmail.java

```
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.divorce.orchestration.tasks;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.Task;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.task.TaskContext;
+import uk.gov.hmcts.reform.divorce.orchestration.service.EmailService;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_CASE_REFERENCE;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_PETITIONER_EMAIL;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_PETITIONER_FIRST_NAME;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.D_8_PETITIONER_LAST_NAME;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.NOTIFICATION_ADDRESSEE_FIRST_NAME_KEY;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.NOTIFICATION_ADDRESSEE_LAST_NAME_KEY;
+import static uk.gov.hmcts.reform.divorce.orchestration.domain.model.OrchestrationConstants.NOTIFICATION_CASE_NUMBER_KEY;
+
+@Component
+public class SendPetitionerClarificationRequestEmail implements Task<Map<String, Object>> {
+
+    private final EmailService emailService;
+
+    @Autowired
+    public SendPetitionerClarificationRequestEmail(final EmailService emailService) {
+        this.emailService = emailService;
+    }
+
+    @Override
+    public Map<String, Object> execute(final TaskContext context, final Map<String, Object> caseData) {
+
+        final String petitionerEmail = (String) caseData.get(D_8_PETITIONER_EMAIL);
+        if (StringUtils.isNotBlank(petitionerEmail)) {
```

Should never be black right? If it is I would throw an exception or at least log a WARN

---

done